internal/data/withdraw_extra_info.go

```diff
@@ -0,0 +1,79 @@
+package data
+
+import (
+	"context"
+	"hash/crc32"
+
+	"github.com/nervina-labs/cota-nft-entries-syncer/internal/biz"
+	"github.com/nervina-labs/cota-nft-entries-syncer/internal/logger"
+)
+
+const pageSize = 10000
+
+var _ biz.WithdrawExtraInfoRepo = (*withdrawExtraInfoRepo)(nil)
+
+type withdrawExtraInfoRepo struct {
+	data   *Data
+	logger *logger.Logger
+}
+
+func NewWithdrawExtraInfoRepo(data *Data, logger *logger.Logger) biz.WithdrawExtraInfoRepo {
+	return &withdrawExtraInfoRepo{
+		data:   data,
+		logger: logger,
+	}
+}
+
+func (rp withdrawExtraInfoRepo) CreateExtraInfo(ctx context.Context, outPoint string, txHash string, lockScriptId uint) error {
+	outPointCrc := crc32.ChecksumIEEE([]byte(outPoint))
+	if err := rp.data.db.WithContext(ctx).Model(WithdrawCotaNftKvPair{}).Where("out_point_crc = ? AND out_point = ?", outPointCrc, outPoint).Updates(WithdrawCotaNftKvPair{TxHash: txHash, LockScriptId: lockScriptId}).Error; err != nil {
+		return err
+	}
+	return nil
+}
+
+func (rp withdrawExtraInfoRepo) FindAllQueryInfos(ctx context.Context) ([]biz.WithdrawQueryInfo, error) {
```

Comment:
It is recommended to pass `page` and `page_size to the function, and return data in batches.